source/code-snippets/usage-examples/updateMany.js

```diff
@@ -0,0 +1,33 @@
+// ignored first line
+const { MongoClient } = require("mongodb");
+
+// Replace the following with your MongoDB deployment's connection string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+const client = new MongoClient(uri);
+
+async function run() {
+  try {
+    await client.connect();
+
+    const database = client.db("sample_mflix");
+    const collection = database.collection("movies");
+
+    // create a filter to update all movies with a 'G' rating
+    const filter = { rated: "G" };
+    const options = {};
+
+    // increment every document matching the filter with 2 more comments
+    const updateDoc = {
+      $inc: {
+        num_mflix_comments: 2,
+      },
+    };
+    const result = await collection.updateMany(filter, updateDoc, options);
```

Comment:
suggesting to remove options, unless you feel it's valuable to include to illustrate parameters.

Reply:
Good idea - removed